NEWS.rst

```diff
@@ -1,5 +1,13 @@
 .. default-role:: code
 
+Unreleased
+==============================
+
+New Features
+------------------------------
+* `defclass` can now take keyword arguments for metaclass for
+  Python 3 metaclass support.
```

Comment:
"for metaclass for Python 3 metaclass support"? I think this could be worded better.